### PR TITLE
feat: Live preview 2.0 support

### PR DESCRIPTION
**Pull request by Amitkanswal**
## Changes-
-Support for Live Preview 2.0
-Dependencies updated
-Snyk fix
## Story- [[4714]](https://contentstack.atlassian.net/browse/MKT-4714)

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -1,29 +1,27 @@
...
-VITE_CONTENTSTACK_API_HOST= for(NA: api.contentstack.io, EU: eu-api.contentstack.com)
-VITE_CONTENTSTACK_MANAGEMENT_TOKEN = your_management_token
+VITE_CONTENTSTACK_PREVIEW_HOST= rest-preview.contentstack.com
+VITE_CONTENTSTACK_PREVIEW_TOKEN= your_live_preview_token
 VITE_CONTENTSTACK_APP_HOST=app.contentstack.com
 VITE_CONTENTSTACK_LIVE_PREVIEW= true
 VITE_CONTENTSTACK_LIVE_EDIT_TAGS= false
 
+# VITE_CONTENTSTACK_API_HOST=api.contentstack.com
+# VITE_CONTENTSTACK_REGION=eu
```
Comment (MaitriGurey1) [inline .env.sample:18]: is this region value for default ? why is it eu and not us ?

**Review by MaitriGurey1 — APPROVED** [review]

File: svelte.config.js [hunk 2]
```diff
@@ -3,13 +3,16 @@ import preprocess from 'svelte-preprocess';
...
 /** @type {import('@sveltejs/kit').Config} */
 const config = {
-	// Consult https://github.com/sveltejs/svelte-preprocess
-	// for more information about preprocessors
-	preprocess: preprocess(),
-
-	kit: {
-		adapter: adapter(),
-	}
+  // Consult https://github.com/sveltejs/svelte-preprocess
```
Comment (abhishek305) [inline svelte.config.js:6]: do we need this github link ?

File: src/sdk/utils.ts [hunk 3]
```diff
@@ -43,9 +46,9 @@ const setLivePreviewConfig = (): contentstack.LivePreview => {
       'Your LP config is set to true. Please make you have set all required LP config in .env'
```
Comment (abhishek305) [inline src/sdk/utils.ts:46]: sentence has grammar error 
Your LP config is set to true. Please make sure you have set all the required LP config in .env.

**Review by abhishek305 — CHANGES_REQUESTED** [review]

**Review by Contentstack-AnkitaD — APPROVED** [review]

File: .env.sample (hunk 1 above)
Comment (Amitkanswal) [reply .env.sample:19]: EU is an example value and Note is added at the bottom as 'us' is default value

File: svelte.config.js (hunk 2 above)
Comment (Amitkanswal) [reply svelte.config.js:6]: will remove it

File: src/sdk/utils.ts (hunk 3 above)
Comment (Amitkanswal) [reply src/sdk/utils.ts:46]: resolved

**Review by abhishek305 — APPROVED** [review]

**Pull request merged**